lib/puppet/provider/mysql_datadir/mysql.rb

```diff
@@ -6,7 +6,7 @@
   initvars
 
   # Make sure we find mysqld on CentOS
-  ENV['PATH']=ENV['PATH'] + ':/usr/libexec'
+  ENV['PATH']=ENV['PATH'] + ':/opt/rh/mysql55/root/usr/bin:/opt/rh/mysql55/root/usr/libexec:/usr/libexec'
```

Comment:
How does this commit warrant asking about BSD? It adds a path extremely specific to Red Hat Collections, `/opt/rh`.